src/tasks/tasks.repository.ts

```diff
@@ -27,8 +28,15 @@ export class TasksRepository extends Repository<Task> {
     return tasks;
   }
 
-  async createTask(createTaskDto: CreateTaskDto): Promise<Task> {
-    const task = this.create(createTaskDto);
+  async createTask(createTaskDto: CreateTaskDto, user: User): Promise<Task> {
+    const { title, description, status } = createTaskDto;
+
+    const task = this.create({
```

Comment:
- xóa line 32
- line 34 -> 39
```suggestion
    const task = this.create({...createTaskDto, user});
```